From 9783640f136ae7a4e3abef5df70994038e2c1e13 Mon Sep 17 00:00:00 2001
From: Shadi Ammouri <shadi@marvell.com>
Date: Mon, 28 Aug 2017 15:27:01 +0300
Subject: [PATCH 092/386] dts: marvell: musdk: agnic device tree nodes

- The platform entry will be used by the agnic netdev driver.
- The uio entry will be used by the generic platform uio driver
  for exposing:
	1) agnic configuration registers to user-space
	2) msi registers

Change-Id: Ifb6a231e7c4cfb4c2f8d604b7b0709b960151992
Signed-off-by: Shadi Ammouri <shadi@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/43483
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/5868
Reviewed-by: Stefan Chulski <Stefan.Chulski@cavium.com>
Reviewed-by: Liron Himi <liron.himi@marvell.com>
Tested-by: sa_ip-sw-jenkins
[RH: Original patch taken from marvell 88F3720 board support SDK 10.0-PR2003]
Signed-off-by: Ruiqiang Hao <Ruiqiang.Hao@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-ap80x.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-ap80x.dtsi b/arch/arm64/boot/dts/marvell/armada-ap80x.dtsi
index bd310f04464d..45961be442ac 100644
--- a/arch/arm64/boot/dts/marvell/armada-ap80x.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-ap80x.dtsi
@@ -451,6 +451,7 @@
 				dma-coherent;
 				status = "okay";
 			};
+
 			uio_agnic_0 {
 				compatible = "marvell,armada-giu-nic-uio";
 				reg = <0x6F0000 0x1000
-- 
2.17.1

